aspnetcore/fundamentals/minimal-apis/7.0-samples/todo-group/data/Model.cs

```diff
@@ -0,0 +1,15 @@
+namespace Data;
+public class Todo
```

Comment:
Classes generally go in their own files to make them easier to find. So this should probably be in `7.0-samples/todo-group/Data/Todo.cs`.

Reply:
alright would do that